tool/tctl/common/tctl.go

```diff
@@ -451,7 +451,7 @@ func loadConfigFromProfile(ccf *GlobalCLIFlags, cfg *service.Config) (*AuthServi
 	}
 	webProxyHost, _ := c.WebProxyHostPort()
 	idx := client.KeyIndex{ProxyHost: webProxyHost, Username: c.Username, ClusterName: profile.Cluster}
-	key, err := keyStore.GetKey(idx)
+	key, err := keyStore.GetKey(idx, client.WithSSHCerts{})
```

Comment:
Not sure why returning ssh certs is not the default behavior TBH. Seems unexpected.

Reply:
Yes agreed, @andrejtokarcik what do you think? It looks like you added/moved some higher level function which would default to this behavior. 

Additionally it looks like a lot of this logic is duplicated. I think these changes were made at the same time as #5938 so it probably got missed due to timing.

Reply:
Requiring `WithSSHCerts` in this manner was a deliberate decision:

Since SSH certs are now cluster-specific, when they are requested the cluster name should be explicitly specified. This is checked [in general for all `CertOption`s](https://github.com/gravitational/teleport/blob/e11862936700e553bf84ef3e0b79c41373c582fd/lib/client/keystore.go#L269-L273).

On the other hand, there are instances where `GetKey` should work [even when the cluster name is not provided](https://github.com/gravitational/teleport/blob/e11862936700e553bf84ef3e0b79c41373c582fd/lib/client/keyagent.go#L267-L271).

It seemed more robust to require `WithSSHCerts` in conjunction with a cluster name check rather than to have potential attempts to read mangled SSH cert filenames. This should be a positive development from the perspective of code readers/maintainers as well inasmuch as it's now clearer which components of the key are expected to be used after a `GetKey` call.